internal/services/workloads/workloads_sap_virtual_instance_resource.go

```diff
@@ -0,0 +1,1129 @@
+			Type:     pluginsdk.TypeString,
+			Required: true,
+			ForceNew: true,
+			ValidateFunc: validation.StringInSlice([]string{
+				string(sapvirtualinstances.SAPProductTypeECC),
+				string(sapvirtualinstances.SAPProductTypeOther),
+				string(sapvirtualinstances.SAPProductTypeSFourHANA),
+			}, false),
+		},
+
+		"deployment_with_os_configuration": {
+			Type:         pluginsdk.TypeList,
+			Optional:     true,
+			ForceNew:     true,
+			MaxItems:     1,
+			AtLeastOneOf: []string{"deployment_with_os_configuration", "discovery_configuration"},
+			Elem: &pluginsdk.Resource{
+				Schema: map[string]*pluginsdk.Schema{
+					"app_location": commonschema.Location(),
+
+					"os_sap_configuration": {
+						Type:     pluginsdk.TypeList,
+						Required: true,
+						ForceNew: true,
+						MaxItems: 1,
+						Elem: &pluginsdk.Resource{
+							Schema: map[string]*pluginsdk.Schema{
+								"sap_fqdn": {
+									Type:         pluginsdk.TypeString,
+									Required:     true,
+									ForceNew:     true,
+									ValidateFunc: validate.SAPFQDN,
+								},
+
+								"deployer_vm_packages": {
+									Type:     pluginsdk.TypeList,
+									Optional: true,
+									ForceNew: true,
+									MaxItems: 1,
+									Elem: &pluginsdk.Resource{
+										Schema: map[string]*pluginsdk.Schema{
+											"storage_account_id": {
+												Type:         pluginsdk.TypeString,
+												Required:     true,
+												ForceNew:     true,
+												ValidateFunc: commonids.ValidateStorageAccountID,
+											},
+
+											"url": {
+												Type:         pluginsdk.TypeString,
+												Required:     true,
+												ForceNew:     true,
+												ValidateFunc: validation.IsURLWithHTTPorHTTPS,
+											},
+										},
+									},
+								},
+							},
+						},
+					},
+
+					"single_server_configuration": {
+						Type:     pluginsdk.TypeList,
+						Optional: true,
+						ForceNew: true,
+						MaxItems: 1,
+						AtLeastOneOf: []string{
+							"deployment_with_os_configuration.0.single_server_configuration",
+							"deployment_with_os_configuration.0.three_tier_configuration",
+						},
+						Elem: &pluginsdk.Resource{
+							Schema: map[string]*pluginsdk.Schema{
+								"app_resource_group_name": commonschema.ResourceGroupName(),
+
+								"subnet_id": {
+									Type:         pluginsdk.TypeString,
+									Required:     true,
+									ForceNew:     true,
+									ValidateFunc: networkValidate.SubnetID,
+								},
+
+								"virtual_machine_configuration": SchemaForSAPVirtualInstanceVirtualMachineConfiguration(),
+
+								"database_type": {
+									Type:         pluginsdk.TypeString,
+									Optional:     true,
+									ForceNew:     true,
+									ValidateFunc: validation.StringInSlice(sapvirtualinstances.PossibleValuesForSAPDatabaseType(), false),
+								},
+
+								"disk_volume_configuration": SchemaForSAPVirtualInstanceDiskVolumeConfiguration(),
+
+								"is_secondary_ip_enabled": {
```

Comment:
`is` is redundant here 
```suggestion
								"secondary_ip_enabled": {
```